nginx/Dockerfile

```
@@ -10,35 +92,129 @@ RUN set -eux; \
+    MODSEC_DEFAULT_PHASE1_ACTION="phase:1,pass,log,tag:'\${MODSEC_TAG}'" \
+    MODSEC_DEFAULT_PHASE2_ACTION="phase:2,pass,log,tag:'\${MODSEC_TAG}'" \
+    MODSEC_PCRE_MATCH_LIMIT_RECURSION=100000 \
     MODSEC_PCRE_MATCH_LIMIT=100000 \
-    MODSEC_PCRE_MATCH_LIMIT_RECURSION=100000
-
-# CRS specific variables
-ENV USER=nginx \
+    MODSEC_REQ_BODY_ACCESS=on \
+    MODSEC_REQ_BODY_LIMIT=13107200 \
+    MODSEC_REQ_BODY_LIMIT_ACTION="Reject" \
+    MODSEC_REQ_BODY_JSON_DEPTH_LIMIT=512 \
+    MODSEC_REQ_BODY_NOFILES_LIMIT=131072 \
+    MODSEC_RESP_BODY_ACCESS=on \
+    MODSEC_RESP_BODY_LIMIT=1048576 \
+    MODSEC_RESP_BODY_LIMIT_ACTION="ProcessPartial" \
+    MODSEC_RESP_BODY_MIMETYPE="text/plain text/html text/xml" \
+    MODSEC_RULE_ENGINE=on \
+    MODSEC_STATUS_ENGINE="Off" \
+    MODSEC_TAG=modsecurity \
+    MODSEC_TMP_DIR=/tmp/modsecurity/tmp \
+    MODSEC_TMP_SAVE_UPLOADED_FILES="on" \
+    MODSEC_UPLOAD_DIR=/tmp/modsecurity/upload \
+    PORT=80 \
+    NGINX_ALWAYS_TLS_REDIRECT=off \
+    SET_REAL_IP_FROM="127.0.0.1" \
+    REAL_IP_HEADER="X-REAL-IP" \
+    REAL_IP_RECURSIVE="on" \
+    PROXY_TIMEOUT=60s \
+    PROXY_SSL_CERT=/etc/nginx/conf/server.crt \
+    PROXY_SSL_CERT_KEY=/etc/nginx/conf/server.key \
+    PROXY_SSL_DH_BITS=2048 \
+    PROXY_SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
+    PROXY_SSL_CIPHERS="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384" \
+    PROXY_SSL_PREFER_CIPHERS=off \
+    PROXY_SSL_VERIFY=off \
+    PROXY_SSL_OCSP_STAPLING=off \
+    SERVER_NAME=localhost \
+    SSL_PORT=443 \
+    TIMEOUT=60s \
+    WORKER_CONNECTIONS=1024 \
+    LD_LIBRARY_PATH=/lib:/usr/lib:/usr/local/lib \
+    NGINX_ENVSUBST_OUTPUT_DIR=/etc/nginx \
+    # overridden variables
+    USER=nginx \
+    # CRS specific variables
     PARANOIA=1 \
     ANOMALY_INBOUND=5 \
     ANOMALY_OUTBOUND=4 \
     BLOCKING_PARANOIA=1
 
+COPY --from=build /usr/local/modsecurity/lib/libmodsecurity.so.${MODSEC_VERSION} /usr/local/modsecurity/lib/
+COPY --from=build /etc/nginx/modules/ngx_http_modsecurity_module.so /etc/nginx/modules/ngx_http_modsecurity_module.so
+COPY --from=build /usr/local/lib/liblmdb.so /usr/local/lib/
+COPY --from=build /usr/share/TLS/server.key /etc/nginx/conf/server.key
+COPY --from=build /usr/share/TLS/server.crt /etc/nginx/conf/server.crt
+COPY --from=build /usr/share/TLS/dhparam-* /etc/ssl/certs/
+COPY --from=build /etc/modsecurity.d/unicode.mapping /etc/modsecurity.d/unicode.mapping
+COPY --from=build /etc/modsecurity.d/modsecurity.conf /etc/modsecurity.d/modsecurity.conf
+COPY --from=crs_release /opt/owasp-crs /opt/owasp-crs
+COPY src/etc/modsecurity.d/modsecurity-override.conf /etc/nginx/templates/modsecurity.d/modsecurity-override.conf.template
+COPY src/etc/modsecurity.d/setup.conf /etc/nginx/templates/modsecurity.d/setup.conf.template
```

I think you're missing the entrypoint scripts here.
```suggestion
COPY src/etc/modsecurity.d/setup.conf /etc/nginx/templates/modsecurity.d/setup.conf.template
COPY nginx/docker-entrypoint.d/*.sh /docker-entrypoint.d/
```